chainer_setup_build.py

```diff
@@ -375,11 +376,13 @@ def cythonize(extensions, force=False, annotate=False, compiler_directives={}):
     run_command(cython_cmdbase + ['--version'])
 
     cython_cmdbase.extend(['--fast-fail', '--verbose', '--cplus'])
-    for ext in extensions:
-        cmd = list(cython_cmdbase)
+    cmd = list(cython_cmdbase)
```

Comment:
`cython_cmdbase` seems a list from the outset. Why do we apply list function to it?